pkg/proxy/logger/logger.go

```diff
@@ -0,0 +1,55 @@
+package logger
```

Comment:
lets rename this to "netlog." "log" collides and "logger" doesn't feel sufficiently precise.